src/services/visualizationService.ts

```diff
@@ -14,7 +14,8 @@ import {
   IVizStepPropsEdge,
 } from '@kaoto/types';
 import { getRandomArbitraryNumber, truncateString } from '@kaoto/utils';
-import ELK, { ElkExtendedEdge, ElkNode } from 'elkjs';
+// @ts-ignore
```

Comment:
why do we need a ts-ignore?

Reply:
because there are no typings available for Dagre, we would need to provide our own or see if this exists already